src/LiveChat/Api/Rest/RestRequest.php

```diff
@@ -252,6 +264,12 @@ private function setCurlOpts(&$curlHandle)
         curl_setopt($curlHandle, CURLOPT_URL, $this->url);
         curl_setopt($curlHandle, CURLOPT_RETURNTRANSFER, true);
         curl_setopt($curlHandle, CURLOPT_HTTPHEADER, $headers);
+
+        if ($this->proxy && $this->proxyPort)
+        {

```

Comment:
Code formatting: please use `if (condition) {` structure. Note: there is no new line between `)` and `{` signs.